OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -126,16 +131,44 @@ public bool HandleMouseInput(MouseInput mi)
 			else
 				editorWidget.SetTooltip(null);
 
+			// Actor drag.
+			if (mi.Button == MouseButton.Left)
+			{
+				if (mi.Event == MouseInputEvent.Down && underCursor != null && (ActorFreeMove || underCursor == Selection.Actor))
```

Comment:
```suggestion
				if (mi.Event == MouseInputEvent.Down && underCursor != null && ((mi.Modifiers.HasModifier(Modifiers.Shift) || underCursor == Selection.Actor))
```

this is the proper way of implementing it

Reply:
Ohhhhh

Reply:
For visibility it might actually make sense to have a checkbox in select panel that sais something like, Disable Select Area. And have it disabled, only with a tooltip that sais to hold shift